### PR TITLE
Documentation update (with urls)

### PR DESCRIPTION
**Pull request by antonaudzei-outreach**
Contains all the changes in https://github.com/getoutreach/extensibility-sdk/pull/15 plus URI -> URL change.
See [What’s the Difference Between a URI and a URL?](https://danielmiessler.com/study/difference-between-uri-url/#:~:text=The%20terms%20%E2%80%9CURI%E2%80%9D%20and%20%E2%80%9C,as%20HTTPs%20%2C%20FTP%20%2C%20etc.) for justification

### CODE REVIEW

File: src/sdk/Validator.ts [hunk 1]
```diff
@@ -105,12 +105,12 @@ export const validate = (application: Application): string[] => {
...
       application.store.medias.forEach((media) => {
-        if (!media.uri) {
+        if (!media.url) {
           issues.push('Uri value is missing');
         } else {
-          const validUrl = utils.urlValidation(media.uri);
+          const validUrl = utils.urlValidation(media.url);
           if (!validUrl) {
-            issues.push('Uri value is not a valid url. Value: ' + media.uri);
+            issues.push('Uri value is not a valid url. Value: ' + media.url);
```
Comment (sheff146) [inline src/sdk/Validator.ts:113]: And this

File: src/sdk/Validator.ts [hunk 2]
```diff
@@ -105,12 +105,12 @@ export const validate = (application: Application): string[] => {
       );
     } else {
       application.store.medias.forEach((media) => {
-        if (!media.uri) {
+        if (!media.url) {
           issues.push('Uri value is missing');
```
Comment (sheff146) [inline src/sdk/Validator.ts:109]: This error message needs editing as well.

**Review by sheff146 — APPROVED** [review]

File: src/sdk/Validator.ts (hunk 2 above)
Comment (antonaudzei-outreach) [reply src/sdk/Validator.ts]: Agree, fixed

File: src/sdk/Validator.ts (hunk 1 above)
Comment (antonaudzei-outreach) [reply src/sdk/Validator.ts]: Agree, fixed

File: src/legacy/ManifestTranslator.ts [hunk 3]
```diff
@@ -161,7 +161,7 @@ export class ManifestTranslator {
     app.store.categories = [];
 
     app.store.description = firstExt.description;
-    app.store.icon = firstExt.host.icon;
+    app.store.iconUrl = '';
```
Comment (nimal-outreach) [inline src/legacy/ManifestTranslator.ts:164]: Out of curiosity - why not use the V1 manifest icon?

**Review by nimal-outreach — APPROVED** [review]

**Pull request merged**